arch/risc-v/src/bl602/bl602_head.S

```diff
@@ -150,17 +150,17 @@ exception_common:
  *  Name: g_intstackalloc and g_intstackbase
  ************************************************************************************/
 
-#if CONFIG_ARCH_INTERRUPTSTACK > 3
+#if CONFIG_ARCH_INTERRUPTSTACK > 15
   .bss
-  .align  4
+  .align  16
```

Comment:
This should be `.balign`.  In RISC-V `.align X` aligns to power of X boundaries, whereas `.balign X` aligns to X bytes boundaries.